app/src/test/java/org/mozilla/fenix/tabstray/syncedtabs/SyncedTabsIntegrationTest.kt

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.tabstray.syncedtabs
+
+import androidx.navigation.NavController
+import io.mockk.mockk
+import mozilla.components.feature.syncedtabs.view.SyncedTabsView
+import mozilla.components.support.test.robolectric.testContext
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mozilla.fenix.R
+import org.mozilla.fenix.helpers.FenixRobolectricTestRunner
+import org.mozilla.fenix.tabstray.ext.toSyncedTabsListItem
+
+@RunWith(FenixRobolectricTestRunner::class)
+class SyncedTabsIntegrationTest {
+
+    @Test
+    fun `WHEN toSyncedTabsListItem is called THEN return a SyncedTabsListItem_Error with the correct text`() {
```

Comment:
Gonna reuse Christian's advice for naming here: 

> nitnit :) In general it's better to keep the method names out of the test names. Otherwise all test names will be stale if you anyone ever refactors those methods.
> 
> e.g. GIVEN crashes should be reported WHEN user restores tab THEN non-fatal crash report is submitted and tab recovered.

https://github.com/mozilla-mobile/fenix/pull/23010#discussion_r802089174